connectors/vault/user_vault_init.sh

```diff
@@ -22,7 +22,7 @@ source $DIR_ROOT/pkg/policy-compiler/policy-compiler.env
 
 export VAULT_BIN=$DIR_ABSTOOLBIN/vault
 #export USER_VAULT_KAFKA_ASSET_KEY=87ffdca3-8b5d-4f77-99f9-0cb1fba1f73f/01c6f0f0-9ffe-4ccc-ac07-409523755e72
-export USER_VAULT_KAFKA_ASSET_KEY="{\"catalog_id\":\"87ffdca3-8b5d-4f77-99f9-0cb1fba1f73f\",\"asset_id\":\"01c6f0f0-9ffe-4ccc-ac07-409523755e72\"}"
+export USER_VAULT_KAFKA_ASSET_KEY="{\"catalog_id\":\"fb4e5fc8-266c-4cf7-a29f-33e3c9d5d091\",\"asset_id\":\"05df45f2-5927-4534-bdff-0d4b923baeea\"}"
```

Comment:
Why is this keep changing? can't we fix it to something?

Reply:
As we use different assets and CPD instances during different times, we need to update the assetids frequently.

Reply:
Fine an alternative that doesn't require changing the source code frequently, such as adding a parameter to this bash script,